plans/util/aws_agentless_workload_setup.yaml

```diff
@@ -0,0 +1,18 @@
+---
+    post_tenant_setup:
```

Comment:
I think we need to update this deployment name, or do we want to add a step to the existing deployment (in  state)

Reply:
Done